src/cplus_plugin/models/base.py

```diff
@@ -240,6 +239,7 @@ class PriorityLayerType(IntEnum):
 
     DEFAULT = 0
     NPV = 1
+    CONSTANT = 2
```

Comment:
@osundwajeff we can consider removing this since it might not be used in the context of PWLs.